pure-docker/deploy-caddy.sh

```diff
@@ -31,6 +31,6 @@ docker run --detach \
     -p 0.0.0.0:80:80 \
     -p 0.0.0.0:443:443 \
     -v $VOLUME:/caddy-storage \
-    --mount type=bind,source="$(pwd)"/caddy/builtins/http.Caddyfile,target=/etc/caddy/Caddyfile \
+    --mount type=bind,source="$(pwd)"/../caddy/builtins/http.Caddyfile,target=/etc/caddy/Caddyfile \
```

Comment:
I think that these path changes might become problematic. They'll likely copy these over and not notice that `../` is now the wrong path. Maybe instead we could duplicate the config files into the subdirectory as well?

Reply:
Do you think moving this to a separate variable would be more clear? i.e.
```sh
REFERENCE_REPO_ROOT="$(pwd)/.."
...

--mount type=bind,source="REFERENCE_REPO_ROOT"/caddy/foobar
```

Reply:
Hmm, I think that would be less clear potentially - because they don't have these scripts in their repo root. Maybe `../` is not so bad on second thought :)

Reply:
Oh yikes haha

I'll make sure to include a release note in the pure-docker upgrade notes to call out that there are path changes to be expected due to this move!